src/Crowdin.Api/Branches/Branch.cs

```diff
@@ -0,0 +1,29 @@
+
+using System;
+using JetBrains.Annotations;
+using Newtonsoft.Json;
+
+namespace Crowdin.Api.Branches
+{
+    [PublicAPI]
+    public class Branch
```

Comment:
We already have the `Branch` class - `src/Crowdin.Api/SourceFiles/Branch.cs`

Reply:
discussed